packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -28,3 +30,57 @@ impl fmt::Display for AccountId {
         write!(fmt, "{}", &self.0)
     }
 }
+
+/// The metadata required to uniquely identify an ontology element that has been persisted in the
+/// datastore.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+#[serde(rename_all = "camelCase")]
+pub struct PersistedOntologyIdentifier {
+    uri: VersionedUri,
```

Comment:
I think this is causing the CI issue btw
```suggestion
    #[component(value_type = String)]
    uri: VersionedUri,
```

Reply:
Ah, that makes sense, thanks!